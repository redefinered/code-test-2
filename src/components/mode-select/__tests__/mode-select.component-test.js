import React from 'react';
import ModeSelect from '../mode-select.component';

import renderer from 'react-test-renderer';

describe('search-bar tests', () => {
  test('renders correctly', () => {
    const tree = renderer.create(<ModeSelect />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
