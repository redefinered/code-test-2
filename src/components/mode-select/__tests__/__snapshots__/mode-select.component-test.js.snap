// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`search-bar tests renders correctly 1`] = `
<View
  style={
    Object {
      "flexDirection": "row",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "flexDirection": "row",
      }
    }
  >
    <RCTSwitch
      accessibilityRole="switch"
      onChange={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Array [
          Object {
            "height": 31,
            "width": 51,
          },
          Object {
            "backgroundColor": "#3e3e3e",
            "borderRadius": 16,
          },
        ]
      }
      value={false}
    />
    <Text
      style={
        Object {
          "marginLeft": 10,
        }
      }
    >
      View by release date
    </Text>
  </View>
</View>
`;
